DOCS/man/options.rst

```diff
@@ -4876,6 +4876,69 @@ DVB
     An example ``input.conf`` could contain:
     ``H cycle dvbin-channel-switch-offset up``, ``K cycle dvbin-channel-switch-offset down``
 
+DVBV5
+-----
+
+``--dvbv5-adapter=<0-100>``
+    Use the given adapter. Default value: 0.
+
+``--dvbv5-audio_pid=<0-9999>``
+    Select a different audio Packet ID (PID). The default is to use
+    the first audio PID found at the channel-name-file.
+
+``--dvbv5-cc=<country code>``
+    Set the default country to be used by the MPEG-TS parsers, in
+    ISO 3166-1 two letter code. If not specified, the default
+    charset is guessed from the locale environment variables.
+
+``--dvbv5-lna=<-1-1>``
+    Enable, disable or put LNA power in auto mode. Not all frontends
+    support it. Valid values are:
+
+    0 - disable
+
+    1 - enable
+
+    -1 - auto
+
+``--dvbv5-demux=<0-100>``
+    Use the given demux. Default value: 0.
+
+``--dvbv5-diseqc_wait=<0-9999>``
```

Comment:
Options usually don't have underscores

Reply:
Agreed, I copied this from dvbv5-zap.

Fixed in [force-push](https://github.com/mpv-player/mpv/compare/66baa96a2d2e195a676ef849fb138a0b6d1822fa..b788e75a772b0b17b23b959ccfbb6a97a6f3a457)